### DISCUSSION TITLE
No registration key sent after donation for Cyberduck

### DISCUSSION THREAD

**Discussion by saorsastudios** [iterate-ch/cyberduck#15507] [Q&A]
I finally had a few dollars to put toward this app, as I plan on using it quite often this year.  I made my donation on the page that is generated from the Cyberduck app when you close it.  I've received a PayPal receipt, however I never got the email from Cyberduck with the key.  I went to their help area, and looked up my email, and it has not been registered.  I then tried emailing them, 1st from the address on the PayPal receipt, and then from a link on their webpage.  Both were returned as no such address found.  Does anyone have any idea on how to get in touch with these people?  Or should I go through the whole mess with PayPal to get my $$ back?  Thank you.

I should note that I emailed them thinking that maybe I had mis-typed my email address.  I wasn't  terribly concerned until my emails were returned as undeliverable, no such address.  I am confident that this is a legit app, but I'd really like to communicate with someone to get this straightened out.

**Comment by dkocher** [comment]
You can always lookup your registration key at https://cyberduck.io/help#recover to receive it with the email address registered. If you mistyped your email when registering, contact us at support@cyberduck.io.

**Reply by saorsastudios** [reply]
Good morning dkocher, and thank you for responding.  As I mentioned in my post above, I tried both of those options.  My address was not registered, as stated by the recovery search response.  I tried 2 different emails, both using links on Cyberduck web pages.  I've attached the mail server responses below.  (and why was my associated "issue" deleted?)

![Screenshot_20240104_093040_BlueMail](https://github.com/iterate-ch/cyberduck/assets/155598701/0d933f30-03d5-4693-9114-1a578cf65a7e)

![Screenshot_20240104_094252_BlueMail](https://github.com/iterate-ch/cyberduck/assets/155598701/59ccc0b4-8e0b-4f73-aaf6-48a4f8e84467)

**Reply by dkocher** [reply]
The response from our mail server indicates you are sending email using a `From` email header that does not match the domain name of the outgoing SMTP server registered in DNS to allow sending email with this sender domain. Yiou wil lfind further explanation in the link of the email response. Please check your mail server settings.

**Reply by dkocher** [reply]
This is not a bug and therefore your issue was deleted. This ticket system is to report bugs and feature requests.

**Comment by saorsastudios** [comment]
Hi David.

I have no idea what any of that means.  I make jewelry & sell it on my website.  I use cyberduck to upload pictures because the company that hosts my website says it's good.  I'm not technical, nor has my email address ever been denied anywhere else.  I simply want a registration key or my donation back.  That is why I'm here.

**Comment by saorsastudios** [comment]
I understand.   Thank you.
